apps/precursor-chemicals/translations/src/en/pages.json

```diff
@@ -121,14 +121,6 @@
     "header": "Who is completing the application?"
   },
 
-  "discharging-licence-responsibilities": {
-    "header": "Will the regulatory affairs and compliance officer discharge all licence responsibilities?"
-  },
-
-  "extra-application-information": {
```

Comment:
UCD requested a slightly different title for the page, which does not include `(optional)`